ksqldb-parser/src/main/java/io/confluent/ksql/parser/tree/QueryContainer.java

```diff
@@ -33,4 +35,11 @@
    * @return the sink info.
    */
   Sink getSink();
+
+  /**
+   * Return an optional query ID if specified by the user
```

Comment:
let's specify here that this is only intended to be used by `INSERT INTO` - that way we can remove it when we remove `INSERT INTO`

Reply:
Done